### PR TITLE
Added retries to the location service api for download packages - upack

### PR DESCRIPTION
**Pull request by aasim**
**Task name**: DownloadPackageV1

**Description**: Added retries to the location service api

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/1755635?src=WorkItemMention&src-action=artifact_link

**Checklist**:
- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [X] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DownloadPackageV1/package-lock.json [hunk 1]
```diff
@@ -1,1100 +1,8 @@
 {
   "name": "downloadpackage",
   "version": "1.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (ByAgenT) [inline Tasks/DownloadPackageV1/package-lock.json:4]: Lockfile downgraded from v2 to v1? Should this be checked in?

Comment (aasim) [reply Tasks/DownloadPackageV1/package-lock.json:4]: I have no idea why this happened... but we should be checking in the package-lock file. Will check again why this happened, good catch

Comment (aasim) [reply Tasks/DownloadPackageV1/package-lock.json]: was using an older npm and node version... updated

**Review by ByAgenT — APPROVED** [review]

**Comment by zhenghao104** [comment]
The retry fix looks good to me, as we discussed offline, we can add the same retry for Upack task.

**Pull request merged**